resolvetosavelives/healthicons
# Script to grab meta-data from Figma

### Also done in this PR
 - adding tags to the modal
 - adding tags to the search
 - supporting multiple keywords out of order in the search
 - jest + yarn:test + yarn test:ci
 - dotenv-flow

### To test
 - see the changes in the `README.md` file
 - try running `yarn import-meta-data`
 - try running `yarn test`
 - try searching keywords and partial keywords out of order
